packages/compass-import-export/src/export/export-csv.spec.ts

```diff
@@ -0,0 +1,281 @@
+import _ from 'lodash';
+import assert from 'assert';
+import fs from 'fs';
+import path from 'path';
+import { promisify } from 'util';
+import chai from 'chai';
+import sinonChai from 'sinon-chai';
+import chaiAsPromised from 'chai-as-promised';
+import temp from 'temp';
+
+temp.track();
+
+import type { DataService } from 'mongodb-data-service';
+import { connect } from 'mongodb-data-service';
+
+import { fixtures } from '../../test/fixtures';
+
+import type { CSVParsableFieldType } from '../csv/csv-types';
+import { exportCSVFromQuery } from './export-csv';
+import { guessFileType } from '../import/guess-filetype';
+import { analyzeCSVFields } from '../import/analyze-csv-fields';
+import { importCSV } from '../import/import-csv';
+import { importJSON } from '../import/import-json';
+
+import allTypesDoc from '../../test/docs/all-bson-types';
+
+const { expect } = chai;
+chai.use(sinonChai);
+chai.use(chaiAsPromised);
+
+describe('exportCSV', function () {
```

Comment:
In `exportJSON` we have a test for exporting an empty collection, would that be useful to have here as well?